tests/testsuite/local_registry.rs

```diff
@@ -241,17 +240,16 @@ fn multiple_names() {
         .build();
 
     p.cargo("check")
-        .with_stderr(
-            "\
+        .with_stderr_data(str![[r#"
 [LOCKING] 3 packages to latest compatible versions
-[UNPACKING] [..]
-[UNPACKING] [..]
-[CHECKING] [..]
-[CHECKING] [..]
-[CHECKING] foo v0.0.1 ([CWD])
-[FINISHED] [..]
-",
-        )
+[UNPACKING] bar v0.0.1 (registry `[ROOT]/registry`)
+[UNPACKING] baz v0.1.0 (registry `[ROOT]/registry`)
+[CHECKING] ba[..] v0.[..]
+[CHECKING] ba[..] v0.[..]
+[CHECKING] foo v0.0.1 ([ROOT]/foo)
```

Comment:
How come we are doing these globs?  If its unordered should we instead set that property?

Reply:
Yes, to address the ordering issue, I'll use `.unordered()` instead!

Reply:
Updated!